src/index.html

```diff
@@ -1,13 +1,50 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image"><img src="images/imac.jpeg" alt="mac"></div>
+      <div class="card__title">
+        <p class="card__title-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__title-product-code">
+          Product code:
+          <span>195434</span>
+        </p>
+      </div>
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review-text">
+          Reviews:
+          <span>5</span>
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-name">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <button class="main-button" data-qa="hover">buy</button>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML We need it to add some "air" and simplify reading. **But don't add them between parent and child elements.**

![image](https://github.com/mate-academy/layout_product-cards/assets/96138503/fac3d11b-1c3d-44fc-96af-6078fc71d4ca)

https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules